src/ontology/cl-edit.owl

```diff
@@ -22687,6 +22689,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22527717"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_0019032 "intestinal tuft cell"@en)
 EquivalentClasses(obo:CL_0019032 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001277)))
 
+# Class: obo:CL_4030023 (respiratory hillock cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7772-9578>) Annotation(oboInOwl:hasDbXref "PMID:30069044"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32726565"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:33442032"^^xsd:string) obo:IAO_0000115 obo:CL_4030023 "An epithelial, transitional cell type between basal and secretory; located in stratified, non-ciliated structures with high cell turnover in respiratory epithelium. In some mammalian species, this cell type has been noted to express KRT13 and KRT4 and is postulated to play a role in squamous barrier function and immunomodulation."^^xsd:string)
```

Comment:
For defs, I would prefer "A hillock cell that..." as it is asserted as that. Not a huge deal IMO, but would make the def more readable imo, but up to you to decide on this. I will accept the PR and let you decide on the change :)

Reply:
Thank you. Agreed. Would you mind re-approving the PR?